tests/RawProfilerHook/RawProfilerHook.Tests/TestsRawProfilerHookAttach.cs

```diff
@@ -61,7 +63,9 @@ public override void Execute()
             }
         }
 
-        [TestMethod]
+        // TODO: RemoteUnitTestExecutor does not run properly due to strong name signing verification
```

Comment:
> TODO: [](http://example.com/codeflow?start=10&length=6)

Did you add a work item for this?

Reply:
I have one internally; I can create a GH issue too for tracking, although I'm working on it now :-)